packages/view/backend/sirius-components-view-edit/src/main/java/org/eclipse/sirius/components/view/provider/ViewItemProviderAdapterFactory.java

```diff
@@ -821,6 +821,29 @@ public Adapter createTextAreaDescriptionAdapter() {
         return this.textAreaDescriptionItemProvider;
     }
 
+    /**
```

Comment:
Please add new RichTextDescription in org.eclipse.sirius.components.view.provider.FormDescriptionItemProvider.collectNewChildDescriptors(Collection<Object>, Object)